HandsOnTutorial/1-hello-world.fsx

```diff
@@ -1,15 +1,19 @@
 ﻿(*** hide ***)
-#load "credentials.fsx"
-
+#load "Thespian.fsx"
+#load "Azure.fsx"
```

Comment:
Change the file names to ``AzureCluster.fsx`` and ``ThespianCluster.fsx``

Can you make it that to switch between the two you comment out one?